keras_cv/layers/object_detection/roi_sampler.py

```diff
@@ -102,24 +110,25 @@ def call(
                 f"num_rois must be less than `num_sampled_rois` ({self.num_sampled_rois}), got {num_rois}"
             )
         rois = bounding_box.convert_format(
-            rois, source=self.bounding_box_format, target="xyxy"
+            rois, source=self.bounding_box_format, target="yxyx"
         )
         gt_boxes = bounding_box.convert_format(
-            gt_boxes, source=self.bounding_box_format, target="xyxy"
+            gt_boxes, source=self.bounding_box_format, target="yxyx"
         )
         # [batch_size, num_rois, num_gt]
-        similarity_mat = iou.compute_iou(rois, gt_boxes, bounding_box_format="xyxy")
+        similarity_mat = iou.compute_iou(
+            rois, gt_boxes, bounding_box_format="yxyx", use_masking=True
+        )
         # [batch_size, num_rois] | [batch_size, num_rois]
         matched_gt_cols, matched_vals = self.roi_matcher(similarity_mat)
         # [batch_size, num_rois]
         positive_matches = tf.math.equal(matched_vals, 1)
+        self.positives.assign_add(tf.reduce_sum(tf.cast(positive_matches, tf.float32)))
```

Comment:
Didn't see this value get read anywhere in the code. Is it for debug?

Reply:
Yeah, it can be used in train.py to allow users to track the status of training, it is quite helpful in debugging.

Reply:
Done.